routes/delius.js

```diff
@@ -102,4 +102,91 @@ router.get('/offenders/nomsNumber/:nomsNumber/allOffenderManagers', (req, res) =
   res.send(ros)
 })
 
+router.get('/probationAreas', (req, res) => {
+  const response = {
+    content: [
+      { code: 'Bir', description: 'Birmingham' },
+      { code: 'Lon', description: 'London' },
+      { code: 'C03', description: 'Coventry' },
+      { code: 'Nottm', description: 'Nottingham' },
+      { code: 'Shf', description: 'Sheffield' },
+      { code: 'Lds', description: 'Leeds' },
+      { code: 'ABC', description: 'ABC probation area' },
+    ],
+  }
+  res.send(response)
+})
```

Comment:
As we were saying the other day, a nice refactoring exercise here, can remove the switch and duplicate references to `content` by creating some objects representing probation areas. ping me if you want to chat about it

Reply:
refactored